app/i18n/es.json

```diff
@@ -505,6 +505,7 @@
     "seeTransaction": "Supervisar la transacción de apertura",
     "send": "Enviar",
     "title": "Mover dinero",
+    "updateAvailable": "Hay disponible una actualización.\nToca para actualizar ahora",
```

Comment:
@dolcalmi can you take a look at this translation and make sure it reads ok?

Reply:
do you have an screenshot of this? is this a button? maybe is better `Hay una actualización disponible. Toque para actualizar` although "toca", "toque" are not common words for this type of action. 

If you have an example of this screenshot maybe I can give you a better translation

@leesalminen no sé como sea en costa rica, si tutean o no, aún hay una mezcla pero creo que la mayoria de las traducciones estan sin tutear

Reply:
Here's a screenshot of where it is. 

![Simulator Screen Shot - iPhone 13 - 2022-01-11 at 15 30 33](https://user-images.githubusercontent.com/3018359/149024622-5b223fe8-642c-4d95-817f-e8f61b9640ec.png)